src/ObjectMakr.ts

```diff
@@ -226,15 +225,17 @@ export class ObjectMakr implements IObjectMakr {
         // For each name in the current object:
         for (let name in base) {
             if (base.hasOwnProperty(name)) {
-                this.functions[name] = new Function() as IClassFunction;
+                if (!this.functions[name]) {
+                    this.functions[name] = new Function() as IClassFunction;
```

Comment:
Instead of using `new Function()`, it would be better to just have `function () { }`. 

In general, prefer avoiding using the class-ified versions of primitives. `""` over `new String()`, etc.